azext_iot/_help.py

```diff
@@ -576,6 +576,49 @@
         -m {module_name} -j ../mymodtwin.json
 """
 
+helps[
+    "iot hub state"
+] = """
+    type: group
+    short-summary: Download, upload or migrate the state of an IoT Hub.
+"""
+
+helps[
+    "iot hub state export"
+] = """
+    type: command
+    short-summary: Saves the configuration and device information of an IoT Hub to a file.
+    examples:
+    - name: Export the devices in the specified hub to the specified file.
```

Comment:
make sure you fix the wording here too cause you aren't just exporting the devices